otterdog/eclipse-capella.jsonnet

```diff
@@ -371,7 +371,7 @@ orgs.newOrg('eclipse-capella') {
       ],
       web_commit_signoff_required: false,
       workflows+: {
-        enabled: false,
+        default_workflow_permissions: "write",
```

Comment:
I would opt for using read by default whenever actions are enabled for repos that dont have workflows yet.

Reply:
done!